deployment/CP02PMCI_Deploy.csv

```diff
@@ -89,3 +89,12 @@ AR-28,,,,CP02PMCI-WFP01-02-DOFSTK000,10,1,2018-04-11T11:58:00,,CGMCP-02PMCI-0001
 AR-28,,,,CP02PMCI-WFP01-03-CTDPFK000,10,1,2018-04-11T11:58:00,,CGMCP-02PMCI-00010,CGVEH-CWFPCP-10302,CGINS-CTDPFK-00125,40.22652,-70.88887,,104,127,
 AR-28,,,,CP02PMCI-WFP01-04-FLORTK000,10,1,2018-04-11T11:58:00,,CGMCP-02PMCI-00010,CGVEH-CWFPCP-10302,CGINS-FLORTK-01119,40.22652,-70.88887,,104,127,
 AR-28,,,,CP02PMCI-WFP01-05-PARADK000,10,1,2018-04-11T11:58:00,,CGMCP-02PMCI-00010,CGVEH-CWFPCP-10302,CGINS-PARADK-20465,40.22652,-70.88887,,104,127,
+#AR-31,,,,CP02PMCI-RII01-02-ADCPTG010,11,1,2018-10-20T00:00:00,,CGMCP-02PMCI-00011,,CGINS-ADCPTG-19151,40.22,-70.88,,104,125,update post-deployment
+#AR-31,,,,CP02PMCI-SBS01-00-STCENG000,11,1,2018-10-20T00:00:00,,CGMCP-02PMCI-00011,,CGCON-BSTC01-00017,40.22,-70.88,,0,125,update post-deployment
```

Comment:
S/N on config sheet SWE0017. Should ID be CGCON-BSTC01-E0017?

Reply:
Confirmed WFP sensor S/Ns